src/components/RadioButtonGroup/styled/errorText.js

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+import attachThemeAttrs from '../../../styles/helpers/attachThemeAttr';
+import ErrorText from '../../Input/styled/errorText';
+
+const StyledTextError = attachThemeAttrs(styled(ErrorText))`
+    text-align: center;
+`;
```

Comment:
Since there is not use of theme values here remove the attachThemeAttrs. Maybe you can get rid of this styled since ErrorText from Input has an alignSelf prop